stream-chat-android-ui-components/src/main/res/layout/stream_item_image_gallery.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.github.chrisbanes.photoview.PhotoView
```

Comment:
Expecting something else in this layout? Maybe better to use PhotoView as a root layout?